specs/examples/inference/gen_perlin_noise.cue

```diff
@@ -0,0 +1,73 @@
+#DST_PATH: "gs://tmp_2w/examples/perlin"
+
+#CHUNK_SIZE: [1024, 1024, 1]
+#DST_INFO_CHUNK_SIZE: [1024, 1024, 1]
+
+#BCUBE: {
+	"@type": "BBox3D.from_coords"
+	start_coord: [0, 0, 0]
+	end_coord: [1024, 1024, 10]
+	resolution: [1, 1, 1]
+}
+
+"@type":         "mazepa.execute"
+target: {
+	"@type": "build_chunked_apply_flow"
+	operation: {
+		"@type": "VolumetricCallableOperation"
+		fn: {
+			"@type": "rearrange"
+			"@mode": "partial"
+			data: [
+				{
+					"@type": "rand_perlin_2d_octaves"
+					input: [1024, 1024]
+					res: [8, 8]
+					octaves: 6
+					device: "cuda"
+				},
+				{
+					"@type": "rand_perlin_2d_octaves"
+					input: [1024, 1024]
+					res: [8, 8]
+					octaves: 6
+					device: "cuda"
+				}
+			]
```

Comment:
This is saying -- "for each given bounding cube, rearrange the following data". This phrasing makes it so that the data is generated only once, and then same data is rearranged over and over. 

One way to fix this is to have `rand_perlin_2d_octaves` produce the channel dimension. What do you think?

Reply:
Adding support for multiple channels is a good suggestion, might even boost performance. It doesn't eliminate the need for rearrange, though? Output is `CXY` (or `NCXY`) and CloudVolume expects `CXYZ`.
I guess I need a dedicated module to do the reshaping and function calling, no way to just do it within the cue spec?

Reply:
If perlin noise generates `NCXY`, a working solution would be to apply rearrange as a write post processor.

At the same time, `CXYZ` is a general `zetta_utils` convention. Whenever we have tensor ops that work with more channels than just XY, we always assume `CXYZ` in and out (eg [[masks.py](https://github.com/ZettaAI/zetta_utils/blob/main/zetta_utils/tensor_ops/mask.py)]). To keep up the convention, I think it would be better to have the perlin noise generator return `CXYZ(N)` than `NCXY`.

Alternatively, we could add a wrapper function that takes in a _callable_ that produces XY data and calls it several times to fill the `C` and potentially `Z` dimensions.